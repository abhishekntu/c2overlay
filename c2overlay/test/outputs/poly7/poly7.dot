digraph graphname {
N28 [color="black", label="O0_N43_N28"];
N23 [color="black", label="mul_Imm_96_N32_N23"];
N22 [color="black", label="N31N22_mul_N31_add_N22_N22"];
N21 [color="black", label="N30N39_mul_N30_add_Imm_128_N39_N21"];
N20 [color="black", label="mul_Imm_12_N15_N20"];
N27 [color="black", label="sqr_N36_N27"];
N26 [color="black", label="N35N12_add_Imm_13824_N35_mul_N12_N26"];
N25 [color="black", label="N34N19_mul_N34_add_N19_N25"];
N24 [color="black", label="sub_Imm_24_N33_N24"];
N8 [color="black", label="N27N21_mul_N27_add_N21_N8"];
N9 [color="black", label="N26N20_mul_N26_add_N20_N9"];
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="I1_N2_N2"];
N3 [color="black", label="I2_N3_N3"];
N4 [color="black", label="sub_N41_N4"];
N5 [color="black", label="N40N25_mul_Imm_8_N40_sub_N25_N5"];
N6 [color="black", label="N29N38_mul_N29_add_Imm_27648_N38_N6"];
N7 [color="black", label="N28N37_mul_N28_add_N37_N7"];
N12 [color="black", label="N9N23_add_Imm_24_N9_mul_N23_N12"];
N13 [color="black", label="mul_N5_N13"];
N10 [color="black", label="N24N4_add_Imm_72_N24_mul_Imm_16_N4_N10"];
N11 [color="black", label="N8N42_mul_N8_add_Imm_2304_N42_N11"];
N16 [color="black", label="N11N13_add_Imm_24_N11_mul_Imm_4_N13_N16"];
N17 [color="black", label="N16N7_mul_Imm_-2_N16_add_N7_N17"];
N14 [color="black", label="N6N18_mul_N6_add_N18_N14"];
N15 [color="black", label="mul_N10_N15"];
N18 [color="black", label="sub_Imm_160_N17_N18"];
N19 [color="black", label="mul_Imm_20_N14_N19"];
N23 -> N7;
N22 -> N28;
N21 -> N25;
N20 -> N5;
N27 -> N13;
N27 -> N12;
N26 -> N9;
N25 -> N8;
N24 -> N21;
N8 -> N9;
N9 -> N22;
N1 -> N22;
N1 -> N9;
N1 -> N8;
N2 -> N10;
N2 -> N5;
N2 -> N14;
N2 -> N4;
N2 -> N25;
N2 -> N15;
N2 -> N17;
N2 -> N26;
N3 -> N6;
N3 -> N12;
N3 -> N4;
N3 -> N20;
N3 -> N21;
N3 -> N11;
N3 -> N17;
N3 -> N27;
N3 -> N16;
N3 -> N23;
N3 -> N19;
N4 -> N7;
N5 -> N18;
N6 -> N15;
N7 -> N26;
N12 -> N25;
N13 -> N14;
N10 -> N6;
N11 -> N14;
N16 -> N13;
N17 -> N24;
N14 -> N8;
N15 -> N22;
N18 -> N11;
N19 -> N7;
}
