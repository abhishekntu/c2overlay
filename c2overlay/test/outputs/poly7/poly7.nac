procedure foo(in s32 t,in s32 x,in s32 y,out s32 D_2451)
{
  localvar s32 D_2447;
  localvar s32 D_2448;
  localvar s32 D_2449;
  localvar s32 D_2450;
  localvar s32 t1_1;
  localvar s32 t1_2;
  localvar s32 t1_3;
  localvar s32 t1_4;
  localvar s32 t1_5;
  localvar s32 t1_6;
  localvar s32 t1_7;
  localvar s32 t1_8;
  localvar s32 t1_9;
  localvar s32 t1_10;
  localvar s32 t2_1;
  localvar s32 t2_2;
  localvar s32 t2_3;
  localvar s32 t2_4;
  localvar s32 t2_5;
  localvar s32 t2_6;
  localvar s32 t2_7;
  localvar s32 t2_8;
  localvar s32 t3_1;
  localvar s32 t3_2;
  localvar s32 t3_3;
  localvar s32 t3_4;
  localvar s32 t3_5;
  localvar s32 t3_6;
  localvar s32 t3_7;
  localvar s32 t4_1;
  localvar s32 t4_2;
  localvar s32 t4_3;
  localvar s32 t4_4;
  localvar s32 t5_1;
  localvar s32 t5_2;
  localvar s32 t5_3;
  localvar s32 t6_1;
  localvar s32 t6_2;
  localvar s32 t6_3;
  localvar s32 t7_1;
  localvar s32 t7_2;
  localvar s32 t8_1;
  localvar s32 t8_2;
  localvar s32 t9_1;
  localvar s32 t10_1;
  localvar s32 t11_1;
L0005:
  D_2447 <= mul y,-2;
  t1_1 <= add D_2447,x;
  t1_2 <= sqr y;
  t1_3 <= add y,24;
  t1_4 <= mul y,12;
  D_2448 <= add y,24;
  t1_5 <= mul D_2448,4;
  t1_6 <= mul y,20;
  t1_7 <= sub x,y;
  t1_8 <= mul y,96;
  D_2449 <= add x,72;
  t1_9 <= mul D_2449,16;
  t2_1 <= sub t1_1,24;
  t2_2 <= mul t1_2,t1_3;
  D_2450 <= mul x,8;
  t2_3 <= sub D_2450,t1_4;
  t2_4 <= mul t1_2,t1_5;
  t2_5 <= mul t1_6,t1_7;
  t2_6 <= mul y,t1_9;
  t3_1 <= mul y,t2_1;
  t3_2 <= sub t2_3,160;
  t3_3 <= add t2_5,t1_8;
  t3_4 <= add t2_6,27648;
  t4_1 <= add t3_1,128;
  t4_2 <= mul y,t3_2;
  t4_3 <= add t3_3,13824;
  t4_4 <= mul x,t3_4;
  t5_1 <= mul x,t4_1;
  t5_2 <= add t4_2,2304;
  t5_3 <= mul x,t4_3;
  t6_1 <= add t5_1,t2_2;
  t6_2 <= mul x,t5_2;
  t7_1 <= mul t,t6_1;
  t7_2 <= add t6_2,t2_4;
  t8_1 <= add t7_1,t7_2;
  t9_1 <= mul t,t8_1;
  t10_1 <= add t9_1,t5_3;
  t11_1 <= mul t,t10_1;
  D_2451 <= add t11_1,t4_4;
}
