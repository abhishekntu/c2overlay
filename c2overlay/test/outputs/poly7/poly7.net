.input N1
pinlist: N1
    
.input N2
pinlist: N2
    
.input N3
pinlist: N3
    
.output out:N28
pinlist: N28
    
.clb N23_blk
pinlist: N3 open open open N23 open open open open 
subblock: N23_blk 0 open open open 4 open open open open

.clb N22_blk
pinlist: N9 N1 N15 open N28 open open open open 
subblock: N22_blk 0 1 2 open 4 open open open open
    
.clb N21_blk
pinlist: N3 N24 open open N21 open open open open 
subblock: N21_blk 0 1 open open 4 open open open open

.clb N20_blk
pinlist: N3 open open open N20 open open open open 
subblock: N20_blk 0 open open open 4 open open open open

.clb N27_blk
pinlist: N3 open open open N27 open open open open 
subblock: N27_blk 0 open open open 4 open open open open

.clb N26_blk
pinlist: N7 N2 open open N26 open open open open 
subblock: N26_blk 0 1 open open 4 open open open open

.clb N25_blk
pinlist: N12 N2 N21 open N25 open open open open 
subblock: N25_blk 0 1 2 open 4 open open open open
    
.clb N24_blk
pinlist: N17 open open open N24 open open open open 
subblock: N24_blk 0 open open open 4 open open open open

.clb N8_blk
pinlist: N1 N14 N25 open N8 open open open open 
subblock: N8_blk 0 1 2 open 4 open open open open
    
.clb N9_blk
pinlist: N8 N1 N26 open N9 open open open open 
subblock: N9_blk 0 1 2 open 4 open open open open
    
.clb N4_blk
pinlist: N2 N3 open open N4 open open open open 
subblock: N4_blk 0 1 open open 4 open open open open

.clb N5_blk
pinlist: N2 N20 open open N5 open open open open 
subblock: N5_blk 0 1 open open 4 open open open open

.clb N6_blk
pinlist: N10 N3 open open N6 open open open open 
subblock: N6_blk 0 1 open open 4 open open open open

.clb N7_blk
pinlist: N4 N19 N23 open N7 open open open open 
subblock: N7_blk 0 1 2 open 4 open open open open
    
.clb N12_blk
pinlist: N3 N27 open open N12 open open open open 
subblock: N12_blk 0 1 open open 4 open open open open

.clb N13_blk
pinlist: N16 N27 open open N13 open open open open 
subblock: N13_blk 0 1 open open 4 open open open open

.clb N10_blk
pinlist: N2 open open open N10 open open open open 
subblock: N10_blk 0 open open open 4 open open open open

.clb N11_blk
pinlist: N3 N18 open open N11 open open open open 
subblock: N11_blk 0 1 open open 4 open open open open

.clb N16_blk
pinlist: N3 open open open N16 open open open open 
subblock: N16_blk 0 open open open 4 open open open open

.clb N17_blk
pinlist: N2 N3 open open N17 open open open open 
subblock: N17_blk 0 1 open open 4 open open open open

.clb N14_blk
pinlist: N11 N2 N13 open N14 open open open open 
subblock: N14_blk 0 1 2 open 4 open open open open
    
.clb N15_blk
pinlist: N6 N2 open open N15 open open open open 
subblock: N15_blk 0 1 open open 4 open open open open

.clb N18_blk
pinlist: N5 open open open N18 open open open open 
subblock: N18_blk 0 open open open 4 open open open open

.clb N19_blk
pinlist: N3 open open open N19 open open open open 
subblock: N19_blk 0 open open open 4 open open open open

