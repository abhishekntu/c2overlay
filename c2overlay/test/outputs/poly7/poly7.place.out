Netlist file: test.net   Architecture file: test.arch.xml
Array size: 5 x 5 logic blocks

#block name	x	y	subblk	block number
#----------	--	--	------	------------
N1		0	3	0	#0
N2		6	5	0	#1
N3		0	1	0	#2
out:N28		0	5	0	#3
N23_blk		5	5	0	#4
N22_blk		1	5	0	#5
N21_blk		3	3	0	#6
N20_blk		5	1	0	#7
N27_blk		1	2	0	#8
N26_blk		2	4	0	#9
N25_blk		2	3	0	#10
N24_blk		4	3	0	#11
N8_blk		1	3	0	#12
N9_blk		1	4	0	#13
N4_blk		5	3	0	#14
N5_blk		5	2	0	#15
N6_blk		3	5	0	#16
N7_blk		5	4	0	#17
N12_blk		2	2	0	#18
N13_blk		2	1	0	#19
N10_blk		3	4	0	#20
N11_blk		3	1	0	#21
N16_blk		1	1	0	#22
N17_blk		4	2	0	#23
N14_blk		3	2	0	#24
N15_blk		2	5	0	#25
N18_blk		4	1	0	#26
N19_blk		4	4	0	#27
